docs/topics/api/addons.rst

```diff
@@ -527,3 +527,24 @@ Compatibilty overrides are used within Firefox i(and other toolkit applications
     :>json string results[].version_ranges[].applications[].min_version: minimum version of the application to be disabled in.
     :>json string results[].version_ranges[].applications[].max_version: maximum version of the application to be disabled in.
     :>json string results[].version_ranges[].applications[].guid: Application `guid <https://addons.mozilla.org/en-US/firefox/pages/appversions/>`_.
+
+
+---------------
+Recommendations
+---------------
+
+.. _addon-recommendations:
+
+This endpoint provides recommendations of other addons to install, fetched from the `recommendation service <https://github.com/mozilla/taar>`_.
+Four recommendations are fetched, but only valid, publicly available addons are shown (so max 4 will be returned, and possibly less).
+
+.. http:get:: /api/v3/addons/recommendations/
+
+    :query string guid: Fetch recommendations for this add-on guid.
+    :query string lang: Activate translations in the specific language for that query. (See :ref:`translated fields <api-overview-translations>`)
+    :query string variant: This endpoint is part of an A/B test.  `a` is fetch recommendations from the recommendation service; `b` is return a fixed fallback.
```

Comment:
As per our discussion, the variable name should be `recommended` with a value of `true` or `false`.

Reply:
https://github.com/mozilla/addons-server/pull/8126/commits/06ed8d22dc8ac7c30b4901beaaeb2abaedb86b62